data/data/libvirt/main.tf

```diff
@@ -5,7 +5,8 @@ provider "libvirt" {
 module "volume" {
   source = "./volume"
 
-  image = "${var.tectonic_os_image}"
+  cluster_name = "${var.tectonic_cluster_name}"
```

Comment:
not being used

Reply:
hmm, https://github.com/openshift/installer/commit/57079cde1ee8180bfffbcc83eed17942e92d18dd#diff-3051f6cd2c80dd39db1934750a7838acR2 looks like its a separate module...

Reply:
> hmm, 57079cd#diff-3051f6cd2c80dd39db1934750a7838acR2 looks like its a separate module...

It is.  This code is calling that module, and, for the first time in 57079cde1e, that module is using that variable.

Reply:
> This code is calling that module, and, for the first time in 57079cd, that module is using that variable.

Grounding this in code, removing this line causes the `volume` module to error out:

```console
$ git describe --dirty
v0.3.0-201-g80bb4ed-dirty
$ git diff
diff --git a/data/data/libvirt/main.tf b/data/data/libvirt/main.tf
index 1be4670..78bbc10 100644
--- a/data/data/libvirt/main.tf
+++ b/data/data/libvirt/main.tf
@@ -5,7 +5,6 @@ provider "libvirt" {
 module "volume" {
   source = "./volume"
 
-  cluster_name = "${var.tectonic_cluster_name}"
   image        = "${var.tectonic_os_image}"
 }

$ hack/build.sh
$ openshift-install --dir=wking --log-level=debug create cluster
...
Initializing modules...
- module.volume
  Getting source "./volume"
- module.bootstrap
  Getting source "./bootstrap"

Error: module "volume": missing required argument "cluster_name"


ERROR Failed to read tfstate: open : no such file or directory 
FATAL Error executing openshift-install: failed to fetch Cluster: failed to generate asset "Cluster": failed to run terraform: failed to initialize Terraform: failed to execute Terraform: exit status 1 
```